auto_nag/scripts/triage_owner_rotations.py

```diff
@@ -0,0 +1,107 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from typing import List, Set
+
+from libmozdata.bugzilla import BugzillaComponent
+from requests import HTTPError
+from tenacity import (
+    RetryError,
+    retry,
+    retry_if_exception_message,
+    stop_after_attempt,
+    wait_exponential,
+)
+
+from auto_nag import logger
+from auto_nag.bzcleaner import BzCleaner
+from auto_nag.component_triagers import ComponentName, ComponentTriagers, TriageOwner
+
+
+class TriageOwnerRotations(BzCleaner):
+    def __init__(
+        self,
+        excluded_components: List[str] = [],
```

Comment:
Nit: given that this is empty, perhaps we can remove it and hope we'll never need it.